platform/ecs/metric_test.go

```diff
@@ -60,7 +62,7 @@ func TestGenerateMetric(t *testing.T) {
 	for _, tc := range tests {
 		mock.metadataPath = fmt.Sprintf("taskmetadata/testdata/metadata_%s.json", tc.mode)
 		mock.statsPath = fmt.Sprintf("taskmetadata/testdata/stats_%s.json", tc.mode)
-		g := newMetricGenerator(mock)
+		g := newMetricGenerator(mock, hostinfo.NewMockGenerator(3876802560.0, 8.0, nil))
```

Comment:
[note] these mock values can be any

Reply:
Yes. Actual values are not tested in ECS. On contrast they're tested in k8s, so maybe I can test them as k8s does. Do you think we should?

Reply:
Basically it's better to improve test coverage, but I think ok for now.

Reply:
Okay, I added at https://github.com/mackerelio/mackerel-container-agent/pull/80/commits/fd37298937d4bd56e3a754e085b4778ef39e7c46 and it reveals the hostinfo is not used in current test cases 🧐

Furthermore, as far as I read https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html, CPU / memory limits are always present at least one of container-level or task-level, so this fallback is not necessary in ECS, but I'd like to keep this implementation for now……